Jellyfin.Plugin.Fanart/Providers/SeriesProvider.cs

```diff
@@ -201,7 +212,7 @@ private void PopulateImages(
                         info.CommunityRating = likes;
                     }
 
-                    if (type == ImageType.Thumb && DateTime.Parse(i.Added,  null) < new DateTime(2016,1,8)) {
+                    if (type == ImageType.Thumb && !(DateTime.TryParse(i.Added, out var added) && added >= new DateTime(2016,1,1))) {
```

Comment:
Should we use the WorkingThumbImageDimensions constant here?

Reply:
I moved that date to a constant. I forgot to use it here. Will fix it later